### ISSUE TITLE
Getting error on locate command

### ISSUE THREAD

**Issue by sanbornsen** [G-Two/subarulink#71]
```
sudipta@Sudipta-Sen-W1FJVTJHQX  ~  python3env  subarulink locate                                                                                                                             ✔  4275  09:18:43
Traceback (most recent call last):
  File "/Users/sudipta/python3env/bin/subarulink", line 8, in <module>
    sys.exit(main())
  File "/Users/sudipta/python3env/lib/python3.9/site-packages/subarulink/app/cli.py", line 715, in main
    LOOP.run_until_complete(cli.single_command(args.command, args.vin, cli.config))
  File "/opt/homebrew/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/sudipta/python3env/lib/python3.9/site-packages/subarulink/app/cli.py", line 555, in single_command
    print(f"Longitude:\t{self.car_data['status'].get('longitude')}")
KeyError: 'status'
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x103b4a2e0>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x103b730a0>, 17.977914666)]']
connector: <aiohttp.connector.TCPConnector object at 0x103b4a310>
```

**Comment by sanbornsen** [comment]
Is this `Unclosed connector` going to drain the battery?

**Comment by G-Two** [comment]
No, it won't drain the battery or do anything to affect your car. I'll try to fix the bug soon (I renamed some constants recently and it looks like I didn't incorporate them into the CLI)

**Comment by G-Two** [comment]
Bug should be fixed now in v0.7.5

**Comment by sanbornsen** [comment]
Yeah it's fixed. Thank you.
